samigo/samigo-app/src/webapp/jsf/author/authorSettings.jsp

```diff
@@ -212,17 +212,21 @@
         });
 
         function expandAccordion(iframId){
-			$('.ui-accordion-content').show();
+			$(‘div#jqueryui-accordion > .ui-accordion-content’).show();
 			mySetMainFrameHeight(iframId);
 			$("#collapseLink").show();
 			$("#expandLink").hide();
+			$(“div#jqueryui-accordion > h3.ui-accordion-header > span”).removeClass(“ui-icon-triangle-1-e”).addClass(“ui-icon-triangle-1-s”);
+			$(“div#jqueryui-accordion > h3.ui-accordion-header”).addClass(“ui-accordion-header-active ui-state-active”);
 		}
 
 		function collapseAccordion(iframId){
 			$('.ui-accordion-content').hide();
 			mySetMainFrameHeight(iframId);
 			$("#collapseLink").hide();
 			$("#expandLink").show();
+			$(“div#jqueryui-accordion > h3.ui-accordion-header > span”).removeClass(“ui-icon-triangle-1-s”).addClass(“ui-icon-triangle-1-e”);
```

Comment:
Hi, this quotes are not right, they are french quotes, you need to update them, thanks :)

Reply:
There was both single quotes and double quotes in the functions already, so I just stuck with how they were. To be clear, which ones are French quotes? Should I be changing these to single or double quotes?

Reply:
I think @SedueRey was referring to the rounded-looking quotes `“` in comparison to the straight/vertical quotes `"` that were already there.

Reply:
Oh, I see that now - will fix. Thanks.